include/revng/Support/IntegerSerialization.h

```diff
@@ -0,0 +1,85 @@
+#pragma once
+
+//
+// This file is distributed under the MIT License. See LICENSE.md for details.
+//
+
+#include "llvm/ADT/ArrayRef.h"
+#include "llvm/Support/Endian.h"
+
+#include "revng/ADT/CompilationTime.h"
+#include "revng/ADT/Concepts.h"
+
+namespace detail {
+
+inline constexpr auto IntEndianness = llvm::support::little;
+inline constexpr auto Unaligned = llvm::support::unaligned;
+
+template<SpecializationOf<std::tuple> T>
+inline constexpr size_t computeSize() {
```

Comment:
`packedSize`?

Reply:
I've renamed the public variable to `packedSize`, I think the helper function should still be called `computeSize` since it follows the semantics of a function